frontend/src/constants.js

```diff
@@ -48,6 +48,10 @@ export const URI_ROOM_SERVICE_SOCKET_PATH = '/socket.io/room'
 const PREFIX_ROOM_SERVICE = '/api/v1/room'
 export const URL_ROOM_SERVICE = URI_ROOM_SERVICE + PREFIX_ROOM_SERVICE
 
+export const PEERJS_HOST = 'localhost'

```

Comment:
Can help change this similar to the above? if production it should be '' so that it can use the leetwithfriend hostname

Reply:
@MarcusTXK tagging as you may have missed it, this will break production so would be great if it can be fixed

Reply:
Fixed, thanks for catching it